libmscore/sym.cpp

```diff
@@ -34,9 +34,9 @@ namespace Ms {
 static const int FALLBACK_FONT = 2;       // Bravura
```

Comment:
This can go wrong at least in one way. The FALLBACK_FONT is Bravura. If we change the order, we need to change this index here.